<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVTKRenderWindows</class>
 <widget class="QMainWindow" name="QtVTKRenderWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVTKRenderWindows</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="view2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>540</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="view4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="view3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>540</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="view1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>541</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QRadioButton" name="clickedVtkBoxWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BoxVisible</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="croppedImageButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CropImage</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>521</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="resliceModeCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Oblique Reslice</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="thickModeCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Thick Slab</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="blendModeGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Blend mode</string>
       </property>
       <widget class="QRadioButton" name="radioButton_Min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Intensity Blend</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_Max">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Intensity Blend</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_Mean">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Mean Blend</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>161</width>
         <height>211</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="AddDistance1Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Distance On View 1</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>é¡µ</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDicomDirectory"/>
   <addaction name="actionActionOpenVti"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpenDicomDirectory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/openFolder.png</normaloff>:/Icons/openFolder.png</iconset>
   </property>
   <property name="text">
    <string>OpenDicomDirectory</string>
   </property>
   <property name="toolTip">
    <string>open directory(where dicom images in)</string>
   </property>
  </action>
  <action name="actionActionOpenVti">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/openVti.png</normaloff>:/Icons/openVti.png</iconset>
   </property>
   <property name="text">
    <string>actionOpenVti</string>
   </property>
   <property name="toolTip">
    <string>open dicom image(vti format)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clickedVtkBoxWidget</sender>
   <signal>clicked(bool)</signal>
   <receiver>croppedImageButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
